src/sentry/api/serializers/models/auditlogentry.py

```diff
@@ -16,6 +18,16 @@ def fix(data):
     return data
 
 
+def override_actor_id(user):
```

Comment:
would add a quick comment about this function -- only making the SCIM token name more readable for now.